tests/acceptance/features/loginUsingEmailAddress.feature

```diff
@@ -0,0 +1,41 @@
+@insulated @disablePreviews
+Feature: login users
+
+As an admin
+I want users to login with their email address stored on a LDAP server
+So that they only need to remember one username and password (SSO)
+
+	Scenario: login with default settings
+		When the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: using ldap filter including email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                                                                    |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid)(\|(mailPrimaryAddress=%uid)(mail=%uid)))) |
+		|ldapEmailAttribute |                                                                                          |
+		And the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: using ldapEmailAttribute but loginFilter lacks email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                            |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid))) |
+		|ldapEmailAttribute | mail                                             |
+		And the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
```

Comment:
Should it? The `ldapLoginFilter` alone determines which users are allowed to login.

`ldapLoginFilterMode`, `ldapLoginFilterEmail`, `ldapLoginFilterUsername` and `ldapLoginFilterAttributes` are used to compose the ldapLoginFilter in the wizard. Once saved only the composed `ldapLoginFilter` matters.

The `ldapEmailAttribute` only determines which attribute to use as the email during user sync.